spec/ship_spec.rb

```diff
@@ -7,4 +7,16 @@
     test_ship = Ship.new('cruiser', 3)
     expect(test_ship).to be_instance_of Ship
   end
+
+  describe '#hit' do
+    it 'reduces the health of the ship by 1' do
+      test_ship = Ship.new('cruiser', 3)
+
+      expect(test_ship.health).to eq 3
+
+      test_ship.hit
+
+      expect(test_ship.health).to eq 2
+    end
+  end
```

Comment:
Should this be tested for when it equals zero so it doesn't go negative? Looks like you may already have that.

Reply:
Tests that the health just goes down by 1